index.js

```diff
@@ -1,17 +1,54 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items = this.items.sort((r1, r2) => r1 - r2);
+    this.length = this.items.length;
+  }
 
-  get(pos) {}
+  get(pos) {
```

Comment:
creo que podria ser

`return !this.items[pos] ? throw new Error("OutOfBounds") : this.items[pos]`